libnmstate/nm/ipv4.py

```diff
@@ -15,9 +15,16 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+import logging
+from operator import itemgetter
 import socket
 
 from . import nmclient
+from libnmstate import iplib
+
+IPV4_DEFAULT_GATEWAY_NETWORK = '0.0.0.0/0'
+MAIN_ROUTE_TABLE = 'main'
+MAIN_ROUTE_TABLE_INT = 0
```

Comment:
This should be `ID` instead of `INT` I guess.

Reply:
Done. 
Changed to:

```
MAIN_ROUTE_TABLE_NAME = 'main'
MAIN_ROUTE_TABLE_ID = 0
```